### ISSUE TITLE
Typst example can't work with theme specified

### ISSUE THREAD

**Issue by AbstractiveNord** [nixon-voxell/bevy_motiongfx#37]
Works only if I remove set theme. Also, I am using Linux and path looks strange, the `.\\` part. Can't set absolute path either.
```
WARN naga_oil::compose::parse_imports: item list imports are deprecated, please use `rust::style::item_imports` (or use feature `allow_deprecated`)`
| #import bevy_render::view View

[
    SourceDiagnostic {
        severity: Error,
        span: Span(
            37906033158448,
        ),
        message: "file not found (searched at .\\assets/themes/Monokai Pro.tmTheme)",
        trace: [],
        hints: [],
    },
]
```

**Comment by nixon-voxell** [comment]
Hmm this is strange, I have encountered many issues regarding this theme path. I will take a look into it. I am on Windows tho, maybe that is the difference. Will work on the path issue also.

**Comment by AbstractiveNord** [comment]
I've got the culprit. [right there](https://github.com/nixon-voxell/bevy_motiongfx/blob/main/crates/motiongfx_typst/src/lib.rs#L82).

**Comment by nixon-voxell** [comment]
Oo thanks, does it work tho if you change it to the correct format? Haha

**Comment by AbstractiveNord** [comment]
Should be fixed in #38 . Yes, it was path problem.

**Comment by nixon-voxell** [comment]
awesome, thanks @AbstractiveNord !
